src/delete/strategies/organization.delete.strategy.ts

```diff
@@ -0,0 +1,18 @@
+import { DeleteStrategy, RestInput } from "../../types"
+import {GitHub} from "@actions/github/lib/utils";
+
+export default class OrganizationDeleteStrategy implements DeleteStrategy {
+  private octokit: InstanceType<typeof GitHub>;
+
+  constructor(octokit: InstanceType<typeof GitHub>) {
+    this.octokit = octokit;
+  }
```

Comment:
It's possible to assign it directly in the constructor signature when using `readonly`:

```suggestion
  constructor(private readonly octokit: InstanceType<typeof GitHub>) {}
```

Reply:
Didn't know this - I'm mainly a .net dev, and can get by in Typescript - thanks!